lib/ts/MemArena.h

```diff
@@ -90,43 +128,36 @@ class MemArena
    */
   MemArena &thaw();
 
-  /** MemArena& empty
+  /** Release all memory.
 
       Empties the entire arena and deallocates all underlying memory. Next block size will be equal to the sum of all
       allocations before the call to empty.
    */
-  MemArena &empty();
+  MemArena &clear();
```

Comment:
Should this be erase()?

Reply:
I don't think so, because `erase` is normally used to erase a particular thing or range, whereas `clear` means "delete everything". E.g. [std::map::clear](http://en.cppreference.com/w/cpp/container/map/clear).